src/js/components/Marathon.jsx

```diff
@@ -513,7 +511,7 @@ var Marathon = React.createClass({
           <div className="container-fluid">
             <div className="navbar-header">
               <a className="navbar-brand" href={window.location.pathname}>
-                <img width="160" height="27" alt="Marathon" src="img/marathon-logo.png" />
+                <img width="160" height="27" alt="Marathon" src="dist/img/marathon-logo.png" />
```

Comment:
Please start the Python server in the root-directory of Marathon UI.
The index.htl in the dist directory isn't distributed. 
It's only to make ```npm run serve``` working, so you don't need a server at your own either.